### PR TITLE
[ARTS-330] Gateway - Changing API tests to use gateway endpoints and start ALL services

### PR DESCRIPTION
**Pull request by eladiw**
## Description
Changing API tests to use gateway endpoints and start ALL service

### Checklist:

- [x] Branch name follows convention (feature/arts-#-short-name OR fix/arts-#-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

### CODE REVIEW

**Review by anatbal — APPROVED** [review]

File: api-tests/specs/assign-roles/assign-roles.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const requests = require('../../data/assign-roles/assign-roles')
 const conf = require('../../config/conf')
-const sitesEndpointUri = ':8083/sites';
-const rolesEndpointUri = ':8082/roles';
-const practitionerEndpointUri = ':8081/practitioner'
-let assignRoleEndpointUri = ':8081/practitioner/{personId}/roles'
+const sitesEndpointUri = ':8072/api/sites';
+const rolesEndpointUri = ':8072/api/roles';
+const practitionerEndpointUri = ':8072/api/practitioner'
+let assignRoleEndpointUri = ':8072/api/practitioner/{personId}/roles'
```
Comment (tamirkamara) [inline api-tests/specs/assign-roles/assign-roles.js:6]: won't it be easier/simpler for the gateway to be on 8080?

**Review by tamirkamara — CHANGES_REQUESTED** [review]
Looks good, although not sure about the port... can we change it?

File: docker-compose.yml [hunk 2]
```diff
@@ -47,6 +47,9 @@ services:
       JDBC_DRIVER: com.microsoft.sqlserver.jdbc.SQLServerDriver
       JDBC_URL: jdbc:sqlserver://sql:1433;databaseName=master;user=sa;password=${SAPASSWORD};loginTimeout=15
       SPRING_PROFILES_ACTIVE: ${PROFILE:-local}
+      EUREKA_CLIENT_SERVICEURL_DEFAULTZONE: "http://discovery-server:8761/eureka/"
+      SPRING_CLOUD_CONFIG_LABEL: "main"
+      SPRING_CLOUD_CONFIG_URI: "http://config-server:8888"
```
Comment (tamirkamara) [inline docker-compose.yml:52]: Why do you need this if you have discovery?

Comment (eladiw) [reply docker-compose.yml:52]: for local (profile) development we dont use discovery....

**Review by anatbal — DISMISSED** [review]

File: api-tests/specs/assign-roles/assign-roles.js (hunk 1 above)
Comment (eladiw) [reply api-tests/specs/assign-roles/assign-roles.js]: changed

**Review by tamirkamara — APPROVED** [review]

**Review by anatbal — APPROVED** [review]

**Pull request merged**